spec/models/user_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'validation test', :aggregate_failures do
+    it 'username presence' do
+      user = User.new(password: 'password', password_confirmation: 'password').save
+      expect(user).to eq(false)
+    end
+
+    it 'password presence' do
+      user = User.new(username: 'nimblehq', password_confirmation: 'password').save
+      expect(user).to eq(false)
+    end
+
+    it 'password confirmation presence' do
+      user = User.new(username: 'nimblehq', password: 'password').save
+      expect(user).to eq(false)
+    end
+
+    it 'password and password confirmation don\'t match' do
+      user = User.new(username: 'nimblehq', password: 'password', password_confirmation: 'password1').save
+      expect(user).to eq(false)
+    end
+
+    it 'username uniqueness' do
+      user1 = User.new(username: 'nimblehq', password: 'password', password_confirmation: 'password').save
+      expect(user1).to eq(true)
```

Comment:
Let move all `expect...` to the end @gutakk

Reply:
Fixed in https://github.com/gutakk/ruby-google-scraper/pull/4/commits/cdff016a8fc702c726f3f13c8ecd273af4636fc8